src/modules/vtol_att_control/vtol_att_control_main.cpp

```diff
@@ -531,6 +531,9 @@ void VtolAttitudeControl::task_main()
 			// update transition state if got any new data
 			if (got_new_data) {
 				_vtol_type->update_transition_state();
+
+				_vtol_type->process_mc_data();
+				fill_mc_att_rates_sp();
```

Comment:
I'm not sure. As it currently stands just calling it right away won't help because it will just overwrite not yet published setpoints. I'm going to follow up with Lorenz and Roman about this. For now I am assuming that during a transition multirotor is the dominant mode so out of the two I picked this.